doc/source/contributing/write_examples.rst

```diff
@@ -0,0 +1,42 @@
+How to write a good example?
+============================
+
+Here are some tips for writing a good example.
+
+Use ``mapdl.convert_script``
+----------------------------
+
+If you want to translate a MAPDL script into a PyMAPDL one, you can first
+run the following commands:: 
+
+    from ansys.mapdl.core import convert_script
+    convert_script(path_MAPDL_script)
+
+.. warning::
+
+    The ``mapdl.convert_script`` function is still on beta state, hence it is advised you check its output.
+    Please, double-check if there are easier ways to write some commands.
```

Comment:
```suggestion
    The ``mapdl.convert_script`` method is still in the beta state. You should check its output.
    There may be easier ways to write some commands.
```
I'd advise that the "For eample, use...." be part of the warning. Otherwise, it's out of context.